podcasts/Onboarding/LoginCoordinator.swift

```diff
@@ -148,6 +150,13 @@ extension LoginCoordinator: SyncSigninDelegate, CreateAccountDelegate {
      }
 
     func signingProcessCompleted() {
+        // This can be called multiple times depending on the flow
+        // With this check we ensure this will be executed only once
+        guard !loginFinished else {
```

Comment:
Good idea, thanks! I changed in another branch just to avoid unnecessary merges and solving conflicts: https://github.com/Automattic/pocket-casts-ios/pull/652/commits/77364f373370adcad2511dd4b94351964a3b1a08